pyoutline/outline/versions/session.py

```diff
@@ -141,6 +140,7 @@ def clean(self):
         Unloads all modules.
         """
         self.__modules.clear()
+        import shutil
```

Comment:
I wish there was a better way to do this - this sort of "global code" setup that runs automatically feels messy to me. Maybe just create an issue for it.

Reply:
I've updated this to no longer reimport the module in the function and instead pass a reference to the shutil module as part of the registration method. This is only required in python 2.7, python 3+ keeps the global reference to the library around.
@bcipriano PTAL.